CAP_PERMON Undeclared on Tegra 4.9.140

PR #201 introduced the use of CAP_PERMON in `perf_event_uncore.c`. Building PAPI on a Tegra 4.9.140 will result in the following error:
```
components/perf_event_uncore/perf_event_uncore.c:652:59: error: ‘CAP_PERFMON’ undeclared (first use in this function); did you mean ‘CAP_SETFCAP’?
                           (cap_data[1].permitted & (1 << (CAP_PERFMON - 32)));
                                                           ^~~~~~~~~~~
```

A possible solution would be to use an `ifdef` to see if `CAP_PERMON` is defined and if it is not we default to use `CAP_SYS_ADMIN`. `CAP_SYS_ADMIN` does the same performance monitoring mechanisms as `CAP_PERFMON` (see [documentation](https://man7.org/linux/man-pages/man7/capabilities.7.html)), but the latter is preferred.

--- a/src/components/perf_event_uncore/perf_event_uncore.c
+++ b/src/components/perf_event_uncore/perf_event_uncore.c
@@ -643,8 +643,12 @@ _peu_init_component( int cidx )
      goto fn_fail;
    }
 
-   perfmon_capabilities = (cap_data[0].permitted & (1 << CAP_SYS_ADMIN)) ||
-                          (cap_data[1].permitted & (1 << (CAP_PERFMON - 32)));
+   #ifdef CAP_PERFMON
+      perfmon_capabilities = (cap_data[0].permitted & (1 << CAP_SYS_ADMIN)) ||
+                             (cap_data[1].permitted & (1 << (CAP_PERFMON - 32)));
+   #else
+      perfmon_capabilities = cap_data[0].permitted & (1 << CAP_SYS_ADMIN);
+   #endif
 
    /* Run the libpfm4-specific setup */
 

